Source/utils/paths.cpp

```diff
@@ -90,8 +90,12 @@ const std::string &ConfigPath()
 
 const std::string &AssetsPath()
 {
+#if __EMSCRIPTEN__
+	assetsPath.emplace("assets/");

```

Comment:
Doesn't this add `assets/` every time?
```suggestion
	if (!assetsPath)
		assetsPath.emplace("assets/");
```

Reply:
Ah good catch! Fixed